docroot/themes/custom/vagovadmin/assets/scss/styles.scss

```diff
@@ -76,8 +76,14 @@
 
 /* Right sidebar on node view pages. */
 
-.view-id-right_sidebar_latest_revision .views-row {
-  padding-bottom: 1em;
+.view-id-right_sidebar_latest_revision {
+  .translate-button {
+    padding-top: 1em;
+  }
+
```

Comment:
Adds space between the Preview button and the Translate button.

Reply:
Nice touch